app/questionnaire/location.py

```diff
@@ -0,0 +1,74 @@
+from flask import url_for
+
+
+class Location(object):
+
+    def __init__(self, group_id, group_instance, block_id):
+
+        self.group_id = group_id
+        self.group_instance = group_instance
+        self.block_id = block_id
+
+    def __eq__(self, other):
+        """
+        Check to see if two locations are equal.
+        Two answers are considered to be equal if their dictionary representations equal one another.
+
+        :param other: An answer to compare
+        :return: True if both answers match, otherwise False.
+        """
+        return isinstance(other, Location) and self.__dict__ == other.__dict__
+
+    def __hash__(self):
+        return hash(self.__dict__.values())
+
+    def __str__(self):
+        """
+        String representation of the location, handy for debug messages
+
+        :return:
+        """
+        return "{}/{}/{}".format(self.group_id, self.group_instance, self.block_id)
+
+    def is_interstitial(self):
+        return self.block_id in ['introduction', 'summary', 'thank-you']
```

Comment:
Agreed offline with @ajmaddaford for this to be looked as part of making interstitials proper blocks.